### DISCUSSION TITLE
Quick white background flash before model is loaded

### DISCUSSION THREAD

**Discussion by ServaisLeon-2TL1** [google/model-viewer#3546] [General]
Hey,

Before my model is loaded there is a quick white screen that appears?
Theses are my CSS configs:
<style>
model-viewer {
  --progress-bar-color: transparent;
}

model-viewer#reveal {
    --poster-color: transparent;
  }

.progress-bar {
  display: none;
  visibility: hidden;
}

</style>

Is there something wrong?

**Comment by elalish** [comment]
We take care of revealing for you. What is `model-viewer#reveal` for? Also, if you're trying to remove the progress bar, just put an empty div in the slot: https://modelviewer.dev/docs/#entrydocs-loading-slots-progressBar

**Comment by henrik-dons** [comment]
Hello,
I'm having a similar issue. On page load, there is an empty box before the model is visible.
So the progress bar is hidden while the model is still not fully loaded.

Link: https://leronza.com/product/iconixbold-iphone-17-pro-and-max-in-24k-gold-rose-gold-platinum-copy/

Can we remove the 1 second flash so that when the loading screen hides, the model is directly visible?
Or is this as good as it gets?
